client/js/jsonedit.js

```diff
@@ -283,7 +283,7 @@ const jeCommands = [
     name: 'convert to nested object',
     context: '^.* ↦ css',
     show:  function() {
-      if(typeof jeStateNow.css != "object")
+      if(typeof jeStateNow.css != "object" || jeStateNow === null)
```

Comment:
I think you want `jeStateNow.css == null` here.

Reply:
Oh. I missed `.css`. Duh.